Add get_indicator_with_dbotscore_unknown function to CommonServerPython (#18293)
demisto/content: Packs/Base/ReleaseNotes/1_18_23.md

#### Scripts
##### CommonServerPython

> - Added the **get_indicator_with_dbotscore_unknown** function to use for cases where the api response to an indicator is not found.

Reviewer: ```suggestion
- Added the *create_indicator_result_with_dbotscore_unknown*. Use this function when an API response to an indicator is not found.
```
